aspnet/dnx/projects.rst

```diff
@@ -1,8 +1,652 @@
+The *shared* section of the *project.json* is designed to allow you to create a project that shares its source with other projects, rather than being compiled to a binary.
+
+The scenario where you would most commonly use this is when you have files such as ``sharedAssemblyInfo.cs`` where you want each project that depends on it to compile that file, rather than depend on a binary that exposes a type for you to consume.
+ 
+.. code-block:: json
+
+    {
+      "shared": "*.cs"
+    }
+
+When you have shared source in your project it will generate a NuGet package with a directory called *shared* containing the shared files. Depending on this package will cause the DNX to compile the code files that are in the shared directory as if they were part of your project.
+
+.. important::
+
+    Because you are adding to the source of a project that depends on your shared code, it is recommended that all the shared code be internal. Having public surface area in the types you are adding to another project is likely to cause problems in the future.
+
+.. note::
+
+    By convention shared project names should end in sources. ``Microsoft.AspNet.Common.Sources``, **not** ``Microsoft.AspNet.Common``.
+
+Per framework compilation
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can target a specific framework to build for your application by using the ``--framework`` option when using the ``build`` command. For example::
+
+	dnu build --framework dnxcore50
+
+Packaging
+---------
+
+You use the ``dnu pack`` command to build NuGet packages to a given directory for your project.
+
+DNU pack
+^^^^^^^^
+
+You can pack a project by specifying the project or default to the current directory. In addition, you can specify the following as options:
+
+- A list of target frameworks to build.
+- A list of configurations to build.
+- The output directory.
+- An output of dependencies.
+- Whether to show output such as the source or destination of the nupkgs.
+- A output of help information.
+
+For example, you can pack the current project based on the current directory::
+
+	dnu pack --out c:\projectOutput
+
+For additional details, run the help command::
+
+	dnu pack --help
+
+Adding package metadata
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Project metadata is information such as the version of your app, author, etc.
+
+To specify this in the *project.json* file you create a key for each of the metadata attributes you care about:
+
+.. code-block:: json
+
+    {
+      "version": "0.1-alpha",
+      "authors": ["John Doe"],
+      "description": "A wonderful library that does nice stuff"
+    }
+    
+* Version: The version of the NuGet package and assemblies generated if you pack/publish your application.
+* Authors: The authors and owners section of the NuGet packages nuspec.
+* Description: The description of the NuGet package.
+
+.. note:: Other metadata can be put into the *project.json* file, but the three above are the only ones that will go into the nuspec of the package that is generated.
```

Comment:
I'm pretty sure there are more metadata fields you can specify. `licenseUrl`, `projectUrl`, etc.